Makefile

```diff
@@ -171,40 +186,42 @@ install-completions:
 	install -m 755 -d ${DESTDIR}${BASHCOMPLETIONSDIR}
 	install -m 644 completions/bash/skopeo ${DESTDIR}${BASHCOMPLETIONSDIR}/skopeo
 
-shell: build-container
+shell:
 	$(CONTAINER_RUN) bash
 
 check: validate test-unit test-integration test-system
 
-# The tests can run out of entropy and block in containers, so replace /dev/random.
-test-integration: build-container
-	$(CONTAINER_RUN) bash -c 'rm -f /dev/random; ln -sf /dev/urandom /dev/random; SKOPEO_CONTAINER_TESTS=1 BUILDTAGS="$(BUILDTAGS)" $(MAKE) test-integration-local'
+test-integration:
+	$(CONTAINER_RUN) $(MAKE) test-integration-local
+
 
-# Intended for CI, shortcut 'build-container' since already running inside container.
-test-integration-local:
+# Intended for CI, assumed to be running in quay.io/libpod/skopeo_cidev container.
+test-integration-local: bin/skopeo
```

Comment:
`hack/make/test-integration` invokes `make bin/skopeo` already — I guess that command in `hack/make/test-integration` can be dropped in favor of adding this dependency, here we have more visible control over the build parameters.

Moving the dependency from shell here is unrelated and non-blocking; I’d prefer not to have _both_ and the potential confusion about how it is expected to work, though.

Reply:
> here we have more visible control over the build parameters.

Yes that was my thought too...did I forget to remove it from the script?

I did, woops.   Okay it's gone now.